app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="public_recipe_container mt-5">
+  <div class="row">
+    <% @recipes.each do |recipe| %>
+      <%= render partial: "recipe", locals: { recipe: } %>
+    <% end %>
+  </div>
+</section>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you improve the design of your pages to make them professional so your application be presentable in your Portfolio.